SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -0,0 +1,293 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+#include "Interop/Connection/SpatialConnectionManager.h"
+#include "CoreMinimal.h"
+
+#define CONNECTIONMANAGER_TEST(TestName) \
+	GDK_TEST(Core, SpatialConnectionManager, TestName)
+
+class FTemporaryCommandLine
+{
+public:
+    FTemporaryCommandLine(const FString& NewCommandLine)
+    {
+    	OldCommandLine = FCommandLine::GetOriginal();
+    	FCommandLine::Set(*NewCommandLine);
```

Comment:
Possibly a big Nit as this is just gonna be used in testing but if multiple of these are created there could be issues.
Perhaps a static FString OriginalCommandLine that only get's set once if the string is empty in the constructor would be preferable.

Reply:
True, it's probably overkill but we might at some point move this class somewhere else and forget about this. Good catch!